core/tests/window/test_window.py

```diff
@@ -1159,3 +1239,63 @@ def test_deprecated_names_closeable():
         match=r"Window.closeable has been renamed Window.closable",
     ):
         assert window.closeable
+
+
+def test_deprecated_full_screen(window, app):
+    """A window can be set full screen using the deprecated API."""
+    full_screen_warning = (
+        "`Window.full_screen` is deprecated. Use `Window.state` instead."
+    )
+    with pytest.warns(
+        DeprecationWarning,
+        match=full_screen_warning,
+    ):
+        assert not window.full_screen
+    with pytest.warns(
+        DeprecationWarning,
+        match=full_screen_warning,
+    ):
+        window.full_screen = True
+    with pytest.warns(
+        DeprecationWarning,
+        match=full_screen_warning,
+    ):
+        assert window.full_screen
+    assert_action_performed_with(
+        window,
+        "set window state to WindowState.FULLSCREEN",
+        state=WindowState.FULLSCREEN,
+    )
+    with pytest.warns(
+        DeprecationWarning,
+        match=full_screen_warning,
+    ):
+        window.full_screen = False
+    with pytest.warns(
+        DeprecationWarning,
+        match=full_screen_warning,
+    ):
+        assert not window.full_screen
+    assert_action_performed_with(
+        window,
+        "set window state to WindowState.NORMAL",
+        state=WindowState.NORMAL,
+    )
+
+    # Setting full screen to False when window is not in full screen, is a no-op
+    #
+    # We cannot assert that the action was not performed, since the same action
+    # was performed previously and would still be in EventLog. Therefore, we
```

Comment:
In which case - you can clear the event log with `Eventlog.reset()`.

Reply:
I have changed it, Thanks.